### DISCUSSION TITLE
Feature Request: command to change shape already drawn.

### DISCUSSION THREAD

**Discussion by volt90** [excalidraw/excalidraw#8277] [General]
I'm a big fan of excalidraw thank you so much for creating this. 

For me, I think it would be helpful to be able to switch from say already drawn rectangle to diamond to circle shape upon a command as I have to change level(acting as heading) of my ideas quite frequently (with my text already written and grouped to shapes).

**Comment by Korayem** [comment]
+1

**Comment by yurtkuran** [comment]
upvote!

**Comment by Mrazator** [comment]
Working on it ATM! Let's track it here https://github.com/excalidraw/excalidraw/issues/8520
